frontend/src/redux/roadmaps/actions.ts

```diff
@@ -340,3 +340,16 @@ export const patchIntegrationConfiguration = createAsyncThunk<
     }
   },
 );
+
+export const notifyUsers = createAsyncThunk<
+  boolean,
```

Comment:
I think return value should be void. The response status code will tell whether the operation was successful.
```suggestion
  void,
```